brevoppskrift-web/frontend/src/routes/template.$templateId.tsx

```diff
@@ -90,28 +106,76 @@ function ContentComponent({ content }: { content: Element }) {
     }
     case ElementType.PARAGRAPH: {
       return (
-        <BodyLong spacing>
+        <BodyLong as="div" spacing>
           {content.paragraph.map((cocs, index) => (
             <ContentOrControlStructureComponent cocs={cocs} key={index} />
           ))}
         </BodyLong>
       );
     }
     case ElementType.PARAGRAPH_TABLE: {
-      return <span>TABLE TODO</span>;
+      return (
+        <Table>
+          <Table.Header>
+            <Table.Row>
```

Comment:
Du kan bruke samme rendering metode for header som de andre radene.

Reply:
Med det så mener jeg at du enten kan lage en ny component for PARAGRAPH_TABLE_ROW som brukes begge steder, eller bare rekursivt bruke komponenten ContentComponent.

Reply:
Eller kanskje ikke, fordi du bruker HeaderCell her :sweat_smile:

Reply:
Det var en refactor jeg tenkte på, men glemte. Men som du påpekte så er det nok enklest å beholde det slik pga HeaderCell